detectron2/data/datasets/pascal_voc.py

```diff
@@ -21,7 +21,7 @@
 # fmt: on
 
 
-def load_voc_instances(dirname: str, split: str):
+def load_voc_instances(dirname: str, split: str, class_names: list):
```

Comment:
Should be `List[str]` with `from typing import List`.
Documentation is missing.